src/ocamlorg_frontend/pages/jobs.eml

```diff
@@ -11,26 +11,37 @@ OCaml community."
             <h1 class="text-title dark:text-dark-title font-bold mb-2">OCaml Job Board</h1>
             <p class="text-content dark:text-dark-content text-xl lg:max-w-[80%]">This is a space where groups, companies, and organisations can advertise their projects directly to the
                 OCaml community.</p>
+               <div class="flex lg:flex-row mt-8 mr-4 max-w-[230px]">
+                <%s!  Hero_section.hero_button ~left_icon:(Icons.edit "w-6 h-6") ~right_icon:(Icons.link "w-5 h-5") ~text:("Add a Job Offer") ~href:("https://github.com/ocaml/ocaml.org/blob/main/CONTRIBUTING.md") "" %>
```

Comment:
Please make sure that the link for contributing always goes to the correct section in the CONTRIBUTING.md document (this is true for all the links that invite people to contribute).

Here, it would be https://github.com/ocaml/ocaml.org/blob/main/CONTRIBUTING.md#content-job